rep-2005.rst

```diff
@@ -50,6 +50,22 @@ To propose changes, such as adding or removing items, make a pull request to tha
 The ensuing public review and discussion will evaluate the change in light of the purpose and scope explained above, with the final merge decision being made by one of the REP maintainers.
 
 
+Schedule of changes
+-------------------
+
+To facilitate understanding of what items make up the integral list of packages for a distribution, accepted changes may only be merged on a fixed schedule, detailed below.
+
+* When the code freeze for distribution A occurs:
+  * the version of this list at that time becomes the *definitive* list for that distribution,
+  * the list becomes open for accepting changes, and
```

Comment:
I am not sure I understand the intention. Doesn't this contradict the previous bullet? If the list "fixed" with the freeze how is it becoming open at the same time?